recipe/meta.yaml

```diff
@@ -1,30 +1,28 @@
-{% set version = "4.2.0" %}
+{% set version = "4.4.0" %}
 
 package:
   name: libtiff
   version: {{ version }}
 
 source:
   - url: https://download.osgeo.org/libtiff/tiff-{{ version }}.tar.gz
-    sha256: eb0484e568ead8fa23b513e9b0041df7e327f4ee2d22db5a533929dfc19633cb
+    sha256: 917223b37538959aca3b790d2d73aa6e626b688e02dcda272aec24c2f498abed
     folder: .
-    patches:
-      - patches/fix_TIFFReadRawStrip_man_page_typo.patch
-      # It seems that a recent patch in tifffile 4.1 may have broken
-      # binary compatibility on windows.
-      # https://gitlab.com/libtiff/libtiff/issues/173
-      # https://github.com/python-pillow/Pillow/issues/4237
-      - patches/use_unix_io.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Maybe someone with more experience can chime in here, but shouldn't this be 0 since this is the first build of the new version? It was 1 before because there was a patch.

Reply:
Yup, this needs to be a `0`.

Reply:
Thanks. for some reason I was on autopilot and was updating the build number when I was making changes in my last commits.